packages/mco/components/create-dr-policy/create-dr-policy.tsx

```diff
@@ -298,62 +205,73 @@ const CreateDRPolicy: React.FC<{}> = () => {
               </HelperText>
             </FormHelperText>
           </FormGroup>
-          {!!odfMCOVersion && (
-            <FormGroup fieldId="policy-name">
-              <Alert
-                className="odf-alert mco-create-data-policy__alert"
-                title={t(
-                  'Data Foundation {{ version }} or above must be installed on the managed clusters to setup connection for enabling replication/mirroring.',
-                  { version: odfMCOVersion }
-                )}
-                variant={AlertVariant.info}
-                isInline
-              />
-            </FormGroup>
-          )}
-          {!!state.selectedClusters.length && (
-            <FormGroup
-              fieldId="selected-clusters"
-              label={t('Selected clusters')}
-            >
-              {state.selectedClusters.map((c, i) => (
-                <SelectedCluster key={c.name} id={i + 1} cluster={c} />
-              ))}
-            </FormGroup>
-          )}
-          <DRReplicationType
-            selectedClusters={state.selectedClusters}
-            replicationType={state.replicationType}
-            syncIntervalTime={state.syncIntervalTime}
-            requiredODFVersion={odfMCOVersion}
-            drPolicies={drPolicies}
-            dispatch={dispatch}
-          />
-          {state.replicationType === REPLICATION_TYPE.ASYNC && (
-            <AdvancedSettings
-              enableRBDImageFlatten={state.enableRBDImageFlatten}
-              dispatch={dispatch}
-            />
-          )}
-          {errorMessage && (
-            <FormGroup fieldId="error-message">
-              <Alert
-                className="odf-alert mco-create-data-policy__alert"
-                title={t('An error occurred')}
-                variant="danger"
-                isInline
-              >
-                {errorMessage}
-              </Alert>
-            </FormGroup>
+          {state.selectedClusters.length === 2 && (
+            <>
+              <FormGroup fieldId="cluster-selection-validation">
+                <SelectedClusterValidation
+                  selectedClusters={state.selectedClusters}
+                  requiredODFVersion={odfMCOVersion}
+                  dispatch={dispatch}
+                />
+              </FormGroup>
+              {state.isClusterSelectionValid && (
+                <>
+                  <FormGroup
+                    fieldId="selected-clusters"
+                    label={t('Selected clusters')}
+                  >
+                    {state.selectedClusters.map((c, i) => (
+                      <SelectedClusterView
+                        key={getName(c)}
+                        id={i + 1}
```

Comment:
will this not work ?
```suggestion
                        id={getName(c)}
```

Reply:
It is an index for display purposes, I will rename to index